pkgs/applications/virtualization/docker/default.nix

```diff
@@ -272,15 +272,15 @@ rec {
   # Get revisions from
   # https://github.com/moby/moby/tree/${version}/hack/dockerfile/install/*
   docker_20_10 = callPackage dockerGen rec {
-    version = "20.10.23";
+    version = "20.10.24";
     cliRev = "v${version}";
-    cliHash = "sha256-fNaRpstyG90Jzq3+U2A42Jj+ixb+m7tXLioIcsegPbQ=";
+    cliHash = "sha256-Q3J5xJ0k70s56dl+Nxp9mc7ig6F9A2cbRNvwfLvpjzo=";
     mobyRev = "v${version}";
-    mobyHash = "sha256-nBPw/M4VC9XeZ9S33HWdWSjY2J2mYpI/TPOzvLjSmJM=";
+    mobyHash = "sha256-7s9RfLsw88aCsHr1ldjTU2Dx66Edv51Flh4uIsmsBYY=";
     runcRev = "v1.1.4";
```

Comment:
This should be updated to 1.1.5

Reply:
Are you sure? On the 20.10.14 tag the version 1.1.4 seems to be used https://github.com/moby/moby/blob/v20.10.24/hack/dockerfile/install/runc.installer

Reply:
thats in the 20.10.25 release

Reply:
I think the changelog is incorrect. Anyway I did not see the 20.10.25 release because it is not listed on the release notes page... I'm going to update the patch to use 20.10.25.